xml/vt_best_practices.xml

```diff
@@ -2805,7 +2800,8 @@ virtio-net-pci.any_layout=on/off
     a better entropy in the &vmguest;.
    </para>
    <para>
-    To use the <filename>hwrng</filename>, add the device in the XML
+    To use the <filename>hwrng</filename>, add an <literal>RNG</literal> device
```

Comment:
Please explain RNG at the beginning of this section:

Virtio RNG (random number generator) is a paravirtualized device ...

and then replace

<filename>hwrng</filename> with "Virtio RNG"